part_2/second_split/method_v2.R

```diff
@@ -0,0 +1,398 @@
+rm(list = ls())
+setwd("/Volumes/GoogleDrive/My Drive/alsta_analysis_LA/second_split")
+
+######################################
+##### Section 1: Import packages #####
+######################################
+library(forecast)
+library(bsts)
+library(glmnet)
+library(plyr)
+library(lme4)
+library(strucchange)
+library(changepoint)
+library(mice)
+source("functions_v2.R")
+
+####################################
+##### Section 2: Preprocessing #####
+####################################
+
+### Impute data ###
+# Remove future data.
+end_date <- as.POSIXlt("2018-06-21", tz = "America/Los_Angeles")
+
+# Impute range: 2017-08-01 to 2017-09-01.
+# Read the original data to get the memory information.
+data_agg_sum <- readRDS("data_agg_sum.rds")
```

Comment:
Can you add a copy of this file to our shared folder?

Reply:
Sure! I'll share you a new folder with all files.